test/container-images/Makefile

```diff
@@ -0,0 +1,27 @@
+DOCKER=docker
+DOCKER_REGISTRY=docker.io
+DOCKER_ORG=atomictests
+
+# $(1) - image name
+# $(2) - image tag
+# $(3) - Dockerfile directory
+define build_push_image
+	@echo Building atomic test image: $(1)
+	$(DOCKER) build -t $(1):$(2) $(3)
+	$(DOCKER) tag $(1):$(2) $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(1):$(2)
+	$(DOCKER) tag $(1):$(2) $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(1):latest
+	$(if $(DOCKER_USER),\
```

Comment:
@ganeshmurthy not sure if the same credentials defined on Travis can be used with docker.io.
Let me know if you think we should use different environment variables for building the pushing test images